views/post_a_peep.erb

```diff
@@ -0,0 +1,8 @@
+Post a Peep
+<form action="/name" method="post">
+
+    <label for="name">Your peep:</label><br>
+    <input type="text" id="peep" name="peep"><br>
+    <input type="submit" value="Submit">
```

Comment:
I noticed when I clicked 'Submit' that it currently goes to '/name' but I can't see it referenced in your code. Could you explain what would have happened on the next page?